ev3dev2/motor.py

```diff
@@ -355,7 +355,7 @@ class Motor(Device):
 
     def __init__(self, address=None, name_pattern=SYSTEM_DEVICE_NAME_CONVENTION, name_exact=False, **kwargs):
 
-        if platform in ('brickpi', 'brickpi3') and not isinstance(self, LargeMotor):
+        if platform in ('brickpi', 'brickpi3') and type(self).__name__ != 'Motor' and not isinstance(self, LargeMotor):
```

Comment:
This was the cleanest solution I could come up with :( You can't use ``isinstance(self, Motor)`` because that will return True for any child class of ``Motor``.

Will go ahead and merge this is...we can always tweak later if we come up with a cleaner solution.